README.md

```diff
@@ -19,12 +19,46 @@ In order to create a trigger that reacts when messages are posted to a Message H
 
 While this list of parameters may seem daunting, they can be automatically set for you by using the package refresh CLI command:
 
-```wsk package refresh```
+1. Create an instance of Message Hub service under your current organization and space that you are using for OpenWhisk.
+
+2. Verify that the the topic you want to listen it already exists in Message Hub or create a new topic to listen for messages, like `mytopic`.
+
+3. Refresh the packages in your namespace. The refresh automatically creates a package binding for the Message Hub service instance that you created.
+
+  ```
+  $ wsk package refresh
+  ```
+  ```
+  created bindings:
+  Bluemix_Message_Hub_Credentials-1
+  ```
+
+  ```
+  $ wsk package list
+  ```
+  ```
+  packages
+  /myBluemixOrg_myBluemixSpace/Bluemix_Message_Hub_Credentials-1 private
+  ```
+
+  Your package binding now contains the credentials associated with your Message Hub instance.
+
+4. Now all you need is to create a Trigger to be fire when new messages are posted to your Message Hub.
```

Comment:
"all you need **to do** is create a Trigger ~to be fire~ that will be fired when new messages are posted to your Message Hub **topic**."